.eslintrc.js

```diff
@@ -314,6 +314,7 @@ module.exports = {
     // Custom rules from eslint-plugin-node-core
     'node-core/no-unescaped-regexp-dot': 'error',
     'node-core/no-duplicate-requires': 'error',
+    'node-core/prefer-proto': 'error',
```

Comment:
This should probably go in `lib/.eslintrc.yml`, or support `Object.create(null)` as well as `ObjectCreate(null)` (because primordials are used only in `lib/`)

Reply:
aha, sounds good, i'll extend it to support both patterns